README.md

```diff
@@ -8,10 +8,17 @@ Every pull request will be checked against WPCS trough github actions.
 
 ## Version History
 
-### 1.3.1
+### 2.0.0
+* Added support for new version
+* Added a checkbox to disable new version
+
+### 1.3.2
```

Comment:
Was 1.3.2 ever released? The release is not listed in the README on the master branch, at least

Reply:
... if not, then move the bullets under here up to the 2.0.0 release